public/sanda/practice/sanda4.html

```diff
@@ -6,22 +6,26 @@
 </head>
 <body>
 
-
 <script>
-    function myFunction1() {
-        console.log('Hello humans!')
+
+    function currentDate() {
+        console.log(new Date().toLocaleDateString());
     }
 
-    function myFunction2(x) {
-        console.log(x)
+    function getPowFive(numb) {
+        console.log(Math.pow(numb, 5));
     }
 
-    function myFunction3() {
-        myFunction1();
-        myFunction2(87);
+    function getSumAndDivide(a, b) {
+        console.log(a + b)/b;
     }
+        let a = prompt("a = ", " "), b = prompt("b = ", " ");
```

Comment:
Please, remove this line, because there is no need to wait for user input, - it is required to have only functions' definitions in the script, according to the task.